tests/dflowfm/test_tim.py

```diff
@@ -13,30 +13,30 @@
     test_reference_dir,
 )
 
-triple_data_for_timeseries_floats = {
-    10: [1.232, 2.343, 3.454],
-    20: [4.565, 5.676, 6.787],
-    30: [1.5, 2.6, 3.7],
-}
 
-single_data_for_timeseries_floats = {
-    0.000000: [0.0000000],
-    10.000000: [0.0100000],
-    20.000000: [0.0000000],
-    30.000000: [-0.0100000],
-    40.000000: [0.0000000],
-    50.000000: [0.0100000],
-    60.000000: [0.0000000],
-    70.000000: [-0.0100000],
-    80.000000: [0.0000000],
-    90.000000: [0.0100000],
-    100.000000: [0.0000000],
-    110.000000: [-0.0100000],
-    120.000000: [0.0000000],
-}
+class TestTimSerializer:
+    triple_data_for_timeseries_floats = [
+        {"time": 10, "data": [1.232, 2.343, 3.454]},
+        {"time": 20, "data": [4.565, 5.676, 6.787]},
+        {"time": 30, "data": [1.5, 2.6, 3.7]},
+    ]
 
+    single_data_for_timeseries_floats = [
```

Comment:
Do you really need this many entries? Can't we shorten the testdata to make it more readable? (And to make the file smaller)

Reply:
this is the example I got when I started on the tim model